tests/integ_tests/inference/native_inference/embedding_models/test_large_hf_model_encode.py

```diff
@@ -110,7 +110,7 @@ def test_hf_text_normalized(self):
             pipeline_embedding = pipeline_embeddings[i]
             self.assertEqual(raw_embedding.shape, pipeline_embedding.shape)
             self.assertTrue((raw_embedding - pipeline_embedding < self.eps).all())
-            self.assertTrue(raw_embedding.shape[0], self.model.model_properties.dimensions)
+            self.assertEqual(raw_embedding.shape[0], self.model.model_properties.dimensions)
```

Comment:
How did this test pass before, if we were providing 2 arguments for `assertTrue?`

Reply:
I think the first is used for a True assertion, which is always true. The second one is used for error message